src/app/app.module.ts

```diff
@@ -103,7 +108,8 @@ const appInitializerFn = (appConfig: AppConfigService) => {
     MessageResolver,
     SociaMediaPostListResolver,
     AuthGuard,
```

Comment:
Maybe sort those

SociaMediaPostListResolver,
SocialMediaRenderService,

AuthService,
AuthGuard,
RoleGuard

Reply:
I think the order matters for at least the first few because the initializer needs to be at the top. I sorted them into categories and alphabetical